### PR TITLE
Lerna fix

### PR DESCRIPTION
**Pull request by milind009**
Bug: The path expecting while identifying various workspace packages for yarn is of the form rootWorkspacesFolder/packageFolderName for e.g if the rootWorkspacesFolder is /packages then package name is midgard-scripting-utils then the expected path is packages/midgard-scripting-utils. But it is only sending back the package name as the path in case of azure.

### CODE REVIEW

File: bin/dry-run.rb [hunk 1]
```diff
@@ -353,7 +353,7 @@ def create_user_npmrc
 
   File.delete(npmrc_path) if File.exist?(npmrc_path)
 
-  npmrc = $fetcher.npmrc_content.split("\n")
+  npmrc = $fetcher.npmrc_content.gsub("\r\n", "\n").split("\n")
   registries = []
```
Comment (GiriB) [inline bin/dry-run.rb:357]: This is required for windows filetypes? After this it should work irrespective whether it's a windows file or a linux file, correct?

File: common/lib/dependabot/file_fetchers/base.rb [hunk 2]
```diff
@@ -242,7 +242,7 @@ def _azure_repo_contents(path, commit)
 
           OpenStruct.new(
             name: File.basename(entry.fetch("relativePath")),
-            path: entry.fetch("relativePath"),
+            path: path + "/" + entry.fetch("relativePath"),
```
Comment (GiriB) [inline common/lib/dependabot/file_fetchers/base.rb:245]: Interesting. Does this affect any other workflows? (like the ones that were running until now?)

File: bin/dry-run.rb (hunk 1 above)
Comment (milind009) [reply bin/dry-run.rb:357]: yes true

File: common/lib/dependabot/file_fetchers/base.rb (hunk 2 above)
Comment (milind009) [reply common/lib/dependabot/file_fetchers/base.rb:245]: No the path value is only used while checking for workspace package jsons

File: bin/dry-run.rb (hunk 1 above)
Comment (milind009) [reply bin/dry-run.rb:357]: Yes that is the change

**Pull request merged**